src/DynamoCoreWpf/Extensions/IViewExtension.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Dynamo.Extensions;
+
+namespace Dynamo.Wpf.Extensions
+{
+    /// <summary>
+    /// An extension to the UI layer of Dynamo.
+    /// </summary>
+    public interface IViewExtension : IDisposable
```

Comment:
All the discussions from Extensions apply the same here I think, I don't think I'll repeat the comments, as I expect you'll make everything consistent :)